server/routes/maintainingReports/coordinator.ts

```diff
@@ -50,7 +50,7 @@ export default class CoordinatorRoutes {
     const data = await this.reportDetailBuilder.build(res.locals.user.username, report)
     const reportEdits = await this.reviewService.getReportEdits(reportId)
     const hasReportBeenEdited = reportEdits?.length > 0
-    const lastEdit = hasReportBeenEdited ? reportEdits.at(-1) : null
```

Comment:
The call to the DB already returns edit history in descending chronological order so the the fix is to get the first element rather than the last element of the returned array.